<ui version="4.0" >
 <class>DocStatusWidget</class>
 <widget class="QWidget" name="DocStatusWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>18</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="statusInfo" >
     <property name="text" >
      <string>Ln: 22 Col: 43 Sel: 234</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="docInfo" >
     <property name="text" >
      <string>Characters: 123767,  Lines: 2454</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="encInfo" >
     <property name="text" >
      <string>Encoding: System </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
